themes/default/content/templates/kubernetes/helm/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Helm Chart on Kubernetes
+layout: template
+
+meta_desc: The Helm Chart template makes it easy to deploy a web application to a Kubernetes cluster with Pulumi and Helm.
+
+meta_image: meta.png
+
+card_desc: Deploy a web application to a Kubernetes cluster with Pulumi and Helm.
+
+template:
+  prefix: helm-kubernetes
+  dirname: my-k8s-app
+  languages:
+    - typescript
+    - python
+    - go
+    - csharp
+    - yaml
+
+cloud:
+  name: Kubernetes
+  slug: kubernetes
+
+---
+
+The Helm Chart template creates an infrastructure as code project in your favorite language that deploys a Helm chart to an existing cluster using Pulumi. It uses Pulumi's [Native Kubernetes Provider](/registry/packages/kubernetes) to create a new namespace in an existing Kubernetes cluster and a new [Helm chart](/registry/packages/kubernetes/api-docs/helm/v3) that deploys a web application. The template generates a complete infrastructure as code program, including sample application code, to give you a working project out of the box that you can customize easily and extend to suit your needs.
```

Comment:
I would change "deploys a web application" to "deploys an application (in this case, the Nginx ingress controller)"